arbos/arbosState/arbosstate.go

```diff
@@ -193,21 +193,26 @@ func InitializeArbosState(stateDB vm.StateDB, burner burn.Burner, chainConfig *p
 		stateDB.SetCode(precompile, []byte{byte(vm.INVALID)})
 	}
 
+	// may be the zero address
+	initialChainOwner := chainConfig.ArbitrumChainParams.InitialChainOwner
+
 	_ = sto.SetUint64ByUint64(uint64(versionOffset), arbosVersion)
 	_ = sto.SetUint64ByUint64(uint64(upgradeVersionOffset), 0)
 	_ = sto.SetUint64ByUint64(uint64(upgradeTimestampOffset), 0)
-	_ = sto.SetUint64ByUint64(uint64(networkFeeAccountOffset), 0) // the 0 address until an owner sets it
+	if arbosVersion >= 2 {
+		_ = sto.SetByUint64(uint64(networkFeeAccountOffset), util.AddressToHash(initialChainOwner))
+	} else {
+		_ = sto.SetByUint64(uint64(networkFeeAccountOffset), common.Hash{}) // the 0 address until an owner sets it
+	}
```

Comment:
Would be nice to refactor out the set and instead just have the version check over a `initialNetworkFeeAccount` variable